package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
```

Comment:
- [ ] Add `webpack-dev-server` to your dev dependencies. Otherwise one can't run this command after dowloading your code.